releasenotes/notes/remove-assemble-2d5d9cea4ca504f5.yaml

```
@@ -0,0 +1,18 @@
+---
+upgrade_misc:
+  - |
+    The ``assemble`` function and related capabilites (contained in the ``assembler`` module)
+    have been removed from the codebase following their deprecation in Qiskit 1.2. 
+    ``assemble`` was used to generate ``Qobj`` in the context of the
+    deprecated  :class:`.BackendV1` workflow. The conversion is no longer necessary as the
```

```suggestion
    deprecated  ``BackendV1`` workflow. The conversion is no longer necessary as the
```
`BackendV1` is going be removed in 2.0 as well, so no need for the link